app/src/layouts/common/layout/layout.jsx

```diff
@@ -110,9 +107,28 @@ export class Layout extends Component {
                     />
                   )}
                 </div>
-                <div className={cx('page-container')}>{this.props.children}</div>
-              </div>
-            </ScrollWrapper>
+                {children}
+              </>
+            ) : (
+              <ScrollWrapper
+                withBackToTop
+                withFooter
+                resetRequired={this.state.resetScroll}
+                onReset={this.unmarkScrollToReset}
+              >
+                <div className={cx('scrolling-content')}>
+                  <div className={cx('header-container')}>
```

Comment:
Please move header to a variable, as it duplicates two times and will be rendered in any case.